app/sidekiq/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -88,20 +114,58 @@ def perform(submission_id)
 
       private
 
+      def file_body
+        @file_body ||= File.read('lib/evss/disability_compensation_form/bdd_instructions.pdf')
+      end
+
       def upload_bdd_instructions
-        client.upload(file_body, document_data)
+        # NOTE: the ApiProviderFactory implements its own Flipper,
+        # ApiProviderFactory::FEATURE_TOGGLE_UPLOAD_BDD_INSTRUCTIONS, which will be used to iteratively
+        # start sending more BDD Instructions to Lighthouse instead of EVSS
+        # However, since using this factory is new behavior, it is hidden behind this additional Flipper, which will act
+        # as a "kill switch" in the event there are problems with the ApiProviderFactory implementation, so we can
+        # revert back to using the EVSS::DocumentsService directly here
+        if Flipper.enabled?(:disability_compensation_use_api_provider_for_bdd_instructions)
+          api_response = upload_via_api_provider
+
+          if upload_provider.is_a?(LighthouseSupplementalDocumentUploadProvider)
+            create_lighthouse_polling_record(api_response)
```

Comment:
I wondering why we are creating the polling record here outside of the UploadProvider class? Like if we want everything that goes through the `LighthouseSupplementalDocumentUploadProvider` to be polled/have a polling record, I would expect that record creation to go in there.